sources/arch/x86_64/pit.c

```diff
@@ -9,3 +9,14 @@ void pit_initialize(int freq_divisor)
     asm_out8(PIT_DATA_PORT0, divisor & 0xFF);
     asm_out8(PIT_DATA_PORT0, (divisor >> 8) & 0xFF);
 }
+
+void pit_sleep(uint16_t ms)
+{
+    uint16_t wait_val = PIT_FREQUENCY / (ms * PIT_MSEC);
+    
+    asm_out8(PIT_MODE_COMMAND, PIT_CHANNEL1 | PIT_LOWBYTE);
```

Comment:
maybe use a pit_write or a pit_read instead of raw out8

Reply:
I didn't write that code x)
I've just removed magic value

Reply:
`pit_initialize` use the same architecture.
I don't see the need for a new function :thinking: